playground/infrastructure/datastore_client.py

```diff
@@ -44,10 +44,10 @@ class DatastoreClient:
 
     _datastore_client: datastore.Client
 
-    def __init__(self):
+    def __init__(self, namespace: str):
         self._check_envs()
         self._datastore_client = datastore.Client(
-            namespace=DatastoreProps.NAMESPACE, project=Config.GOOGLE_CLOUD_PROJECT
+            namespace=namespace, project=Config.GOOGLE_CLOUD_PROJECT
```

Comment:
I'm not comfortable that we have two related properties of the datastore client but they are obtained from different sources. Maybe it's not too late to reconsider this approach and have both as env variables or arguments? @TSultanov

Reply:
Moved this configuration option from environment variable to the `ci_cd.py` script arguments and updated the ci/cd configs which use this script